Corzbank.Services/Interfaces/ILoggerManager.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Corzbank.Services.Interfaces
+{
+    public interface ILoggerManager
+    {
+        void LogInfo(string message);
+        void LogWarn(string message);
+        void LogDebug(string message);
+        void LogError(string message);
```

Comment:
Add empty lines between methods for consistency

Reply:
resolved